From: =?utf-8?q?Guido_G=C3=BCnther?= <agx@sigxcpu.org>
Date: Mon, 15 Apr 2019 16:32:43 +0200
Subject: Drop failing haiku test

The tests moved to osinfo-db anyway in newer versions
---
 .../haikunightly/haiku-nightly-anyboot.iso.txt     | 29 ----------------------
 1 file changed, 29 deletions(-)
 delete mode 100644 tests/isodata/haiku/haikunightly/haiku-nightly-anyboot.iso.txt

diff --git a/tests/isodata/haiku/haikunightly/haiku-nightly-anyboot.iso.txt b/tests/isodata/haiku/haikunightly/haiku-nightly-anyboot.iso.txt
deleted file mode 100644
index 1031567..0000000
--- a/tests/isodata/haiku/haikunightly/haiku-nightly-anyboot.iso.txt
+++ /dev/null
@@ -1,29 +0,0 @@
-CD-ROM is in ISO 9660 format
-System id: LINUX
-Volume id: bootimg
-Volume set id: 
-Publisher id: 
-Data preparer id: 
-Application id: GENISOIMAGE ISO 9660/HFS FILESYSTEM CREATOR (C) 1993 E.YOUNGDALE (C) 1997-2006 J.PEARSON/J.SCHILLING (C) 2006-2007 CDRKIT TEAM
-Copyright File id: 
-Abstract File id: 
-Bibliographic File id: 
-Volume set size is: 1
-Volume set sequence number is: 1
-Logical block size is: 2048
-Volume size is: 1624
-El Torito VD version 1 found, boot catalog is in sector 32
-Joliet with UCS level 3 found
-Rock Ridge signatures version 1 found
-Eltorito validation header:
-    Hid 1
-    Arch 0 (x86)
-    ID ''
-    Key 55 AA
-    Eltorito defaultboot header:
-        Bootid 88 (bootable)
-        Boot media 3 (2.88MB Floppy)
-        Load segment 0
-        Sys type 0
-        Nsect 1
-        Bootoff 21 33
